.github/workflows/deploy.yml

```diff
@@ -73,7 +73,7 @@ jobs:
   deploy-dev:
     name: Deploy to dev
     needs: [test, build]
-    if: github.ref == 'refs/heads/main'
+    if: github.ref == 'refs/heads/dag-590-unleash'
```

Comment:
Skal denne fortsatt være dag-590-unleash?

Reply:
Nei, det er kun satt slik midlertidig for at vi kan teste det i dev :)